helm-chart/renku-graph/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: '1.0'
 description: A Helm chart for renku graph services
 name: renku-graph
-version: 0.32.0-a16a4c5
+version: 0.31.2-6929c8b
```

Comment:
The versions don't match here. I guess the version bot is going to take care of this after the PR has been merged?

Reply:
Kind of :) Because that's a bug fix I just want a bugfix release for that change which would be `0.31.2` instead of `0.32.0` set by default by the bot.